src/domain/auth/authenticate.ts

```diff
@@ -12,12 +12,15 @@ import {
 } from './state/BackendAuthenticationActions';
 import { TUNNISTAMO_API_TOKEN_ENDPOINT } from '../api/constants/ApiConstants';
 import { BackendTokenResponse } from './types/BackendAuthenticationTypes';
+import { getCurrentLanguage } from '../../common/translation/TranslationUtils';
 
 export const loginTunnistamo = (path?: string) => {
   userManager
-    .signinRedirect(
-      path ? { data: { path: path } } : { data: { path: '/profile' } }
-    )
+    .signinRedirect({
+      data: { path: path || 'profile' },
+      /* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Imported I18n instance doesnt reflect user selected language correctly.
Only i18n from component is.

Reply:
We can only use i18n from component if we are working with a component? I see no FC component around here?

I tested my implementation, and it works correctly also when changing language.

Reply:
Yeah just worry if you need exact correct language, lgtm

Reply:
One more possible corner case bug in here :D 
But it is at least in the same file. And the risk is low and impact negligible.